rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import babel from '@rollup/plugin-babel';
+import resolve from '@rollup/plugin-node-resolve';
+
+export default {
+    input: 'src/index.ts',
+    output: {
+        file: 'dist/bundle.js',
```

Comment:
what about to call it `dist/index.js`, bcz it is set in `package.json/main`

Reply:
Fixed